websites/P/Pixelmon Mod/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "976435781486911509"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const page = document.location.pathname,
```

Comment:
Update all references too
```suggestion
	const pathname = document.location,
```